README.md

```diff
@@ -1 +1,31 @@
-# Flashcard-Maker
\ No newline at end of file
+# Flashcard-Maker
+
+## Deck Creation DSL
+### Example
+```
+Create Deck French with Tags language, hard, midterm using Style mystyle:  
+(1) Front:Back
+(2) Front:Back
+(3) Front:Back
+
+Create Style mystyle:
+Color = Red
+Direction = Horizontal
+Align = Center
+```
+
+### EBNF
+```
+CREATE_DECK ::= “Create Deck” NAME (“with Tags“ TAGS | “using Style” STYLE)*”:” DECK
+DECK ::= CARD+
+CARD ::= “(”NUMBER”) ” STRING “:” STRING
+TAGS ::= STRING”,”? TAGS
```

Comment:
Would this be better? 
TAGS ::= TAG (“, ” TAG)*
TAG ::= [^,]+
This is similar to the example from class:
![image](https://user-images.githubusercontent.com/20777446/82181333-2cf5a080-9897-11ea-84d2-11147138fe64.png)